packages/app/src/App.tsx

```diff
@@ -219,11 +219,11 @@ const routes = (
         <LightBox />
       </TechDocsAddons>
     </Route>
-    <FeatureFlagged with="scaffolder-next-preview">
+    <FeatureFlagged with="scaffolder-legacy">
       <Route
         path="/create"
         element={
-          <NextScaffolderPage
+          <LegacyScaffolderPage
```

Comment:
Is the `NextScaffolderPage` a drop-in replacement for the legacy one? as in will we roll this out as a breaking change or not?

Reply:
Unfortunately the breaking change is not in the types that we own, it's in the FieldExtension types. So yes it's a breaking change, and yes it's a drop in here, but not everywhere.

Reply:
Unfortunately the breaking change is not in the types that we own, it's in the FieldExtension types. So yes it's a breaking change, and yes it's a drop in here, but not everywhere.